src/hotspot/os/aix/os_aix.cpp

```diff
@@ -633,7 +633,7 @@ static void *thread_native_entry(Thread *thread) {
     address low_address = thread->stack_end();
     address high_address = thread->stack_base();
     lt.print("Thread is alive (tid: %zu, kernel thread id: %zu"
-             ", stack [" PTR_FORMAT " - " PTR_FORMAT " (" SIZE_FORMAT "k using %luk pages)).",
+             ", stack [" PTR_FORMAT " - " PTR_FORMAT " (%zuk using %luk pages)).",
```

Comment:
Is `%lu` compatible with multiple platforms or should it also be changed to `%zu`?

Reply:
I think aix only supports a 64 bit platform, so this format specifier is probably right for that.  I'm not going to change pre-existing formatting for aix without being able to test it.